assets/components/pages/project/curate/VariantData.js

```diff
@@ -147,10 +152,240 @@ class VariantData extends React.Component {
     showAll: false,
   };
 
+  getGQFrequencies() {
+    const { variant } = this.props;
+
+    const options = {
+      scales: {
+        x: {
+          type: "linear",
+          beginAtZero: true,
+          max: 100,
+          grid: {
+            display: false,
+          },
+          ticks: {
+            font: {
+              size: 14,
+            },
+            color: "#333",
+          },
+        },
+        y: {
+          grid: {
+            display: false,
+          },
+          ticks: {
+            font: {
+              size: 14,
+            },
+            color: "#333",
+            callback: (value) => (Number.isInteger(value) ? value : null),
+          },
+        },
+      },
+    };
+
+    if (!variant.GQ_all) {
+      // Handle the case where GQ_all is null
+      return { data: { labels: [], datasets: [] }, options };
+    }
+
+    // Calculate the frequency of each GQ_all value
+    const gqFrequencies = variant.GQ_all.reduce((acc, value) => {
+      const bin = Math.floor(value / 5) * 5; // Bin values in groups of 5
+      acc[bin] = (acc[bin] || 0) + 1;
+      return acc;
+    }, {});
```

Comment:
This is a case where the mdn web docs suggest against using `Array.reduce`, instead opting for a `for ... loop`. `.reduce` is usually used to mimic immutable data practices in functional programming, but this is mutated `acc` on each iteration [reduce#when_to_not_use_reduce](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce#when_to_not_use_reduce).

(Ditto for below too)